README.md

```diff
@@ -30,9 +30,39 @@ If you wish to contribute to this template, the following information may be hel
  * cordova: 4.0+
 
 ## Build instructions
- * Edit [fhconfig.json](www/fhconfig.json) to include the relevant information from RHMAP.  
- * cordova serve  
- 
+
+### configuration
+Edit [fhconfig.json](www/fhconfig.json) to include the relevant information from RHMAP.  
+
+### Grunt
+
+This template uses [Grunt](http://gruntjs.com/), the Javascript Task Runner. To use Grunt with this Template App, do the following:
+
+* Install grunt: ```npm install -g grunt-cli```
+* In your App directory, run: ```npm install```. This installs Grunt plugins, etc for use with this App.
+* Run ```grunt serve``` to preview this App locally
+
+
+### Local development
+
+You can also use Grunt to point your App at a local developement server. To do this, use the ```grunt serve:local``` command. Some notes on using the serve:local task:
+
+* by default, the local server development url is: http://localhost:8001
+* you can change this directly in your local Gruntfile.js, in the app config:
+
+```
+  app: {
+    // configurable paths
+    app: 'www',
+    url: '',
+    default_local_server_url: 'http://localhost:8001'
+  },
+```
+
+* you can also pass a 'url' optional flag to server:local, e.g. ```grunt serve:local --url=http://localhost:9000```
+
+* You can also write your own tasks by extending the Gruntfile.js, e.g. add a 'serve:live' target that hits your server in your FeedHenry live enivronment.
+
```

Comment:
do we really want to duplicate this info, or would it be better to link to the documentation?